packages/redirects-utils/src/index.js

```diff
@@ -0,0 +1,84 @@
+const fs = require('fs')
+const path = require('path')
+const { promisify } = require('util')
+
+const writeFileAsync = promisify(fs.writeFile)
+
+const resolveConfig = require('@netlify/config')
+const { getConfigPath } = require('@netlify/config')
+const { parseRedirectsFormat, parseNetlifyConfig } = require('netlify-redirect-parser')
+const configorama = require('configorama')
+
+async function parseFile(parser, name, filePath) {
+  const result = await parser(filePath)
+  if (result.errors.length) {
+    console.error(`Warnings while parsing ${name} file:`)
+    result.errors.forEach(err => console.error(`  ${err.lineNum}: ${err.line} -- ${err.reason}`))
+  }
+  return result.success
+}
+
+async function getAll(projectPath) {
+  const rulesUnique = new Set()
+
+  const configPath = await getConfigPath(undefined, projectPath)
```

Comment:
This is the default behavior in Netlify CLI, I just mimicked that here. Since we're parsing both `_redirects` file and the config file here, I guess it is sufficient. But open to other opinions.